### DISCUSSION TITLE
KeePassXC -  integration with additional SSH clients

### DISCUSSION THREAD

**Discussion by anatol06** [keepassxreboot/keepassxc#9708] [Ideas]
Are there any intentions to integrate KeePassXC with alternative SSH clients? 
From my perspective, it would be quite convenient to have the ability to utilize KeePassXC in conjunction with SSH clients like [Bitvise SSH Client](https://www.bitvise.com/) and [MobaXterm](https://mobaxterm.mobatek.net/).
And I have another suggestion: After locking the DB, I see the "Unlock KeePassXC Database" title on the Unlock page and the db path.
Wouldn't it be better to hide the DB path shown below the title?

**Comment by droidmonkey** [comment]
We don't integrate with ssh clients. We can provide ssh keys to ssh agent implementations like putty and openssh.

**Reply by anatol06** [reply]
Ok, I understood how it works and now I can get the key in Bitvise SSH Client using  OpenSSH. 
In Bitvise:       Client key manager > More > Reload from Agents
In Windows:  Services > double click on OpenSSH > Startup type: Automatic > Start

**Comment by topmask** [comment]
How to use keepassxc ssh agent with MobaXterm. You can use Pageant SSH agent, but you cannot use openssh agent
